torchgen/model.py

```diff
@@ -2532,14 +2536,26 @@ class BaseOperatorName:
     # Doing that is BC-breaking though, so we're stuck with the above modeling.
     functional_overload: bool = False
 
+    # NB: We don't officially support namespace in FunctionSchema, we treat this prefix
+    # as part of the base operator name, for __str__() to consume.
+    # The canonical input (from the rest of the infra) will not contain namespace, but
+    # we have a usecase in ExecuTorch where we want to support BaseOperatorName with namespace.
+    namespace: str = ""
```

Comment:
nit: maybe this should be an `Optional[str]`? That way it's clearer tha you don't ned to do the `namespace::op_name` logic if no namespace was provided